docs-chef-io/content/habitat/on_prem_builder/install/builder_oauth.md

```diff
@@ -103,13 +103,13 @@ With an internet-connected computer, follow these steps:
     export DOWNLOAD_DIR=/path/to/download/directory
 
     hab pkg exec habitat/pkg-sync pkg-sync \
-      --channel stable \
+      --channel base \
```

Comment:
we want this to remain stable until we build and release the builder components against base.